awx/ui_next/_Dockerfile.Temp

```diff
@@ -0,0 +1,6 @@
+# This is not part of the PR - debugging something
```

Comment:
I guess we'd want to remove this or add to .gitignore?

Reply:
no I intend to remove this file before merging, @dsesami are currently debugging something and using this as a tool.